jobmaster/dm/runtime/worker_status.go

```diff
@@ -0,0 +1,39 @@
+package runtime
+
+import (
+	"github.com/hanfei1991/microcosm/lib"
+)
+
+type WorkerStage int
+
+const (
+	WorkerCreating WorkerStage = iota
+	WorkerOnline
+	WorkerFinished
+	WorkerOffline
+	// WorkerDestroying
+)
+
+type WorkerStatus struct {
+	Task  string
+	ID    lib.WorkerID
+	Unit  lib.WorkerType
+	Stage WorkerStage
```

Comment:
this is the expected stage or current stage?

Reply:
worker has no expected stage